docs/learn/learn-account-generation.md

```diff
@@ -133,6 +133,31 @@ message.
 Note that this password does NOT protect your seed phrase. If someone knows the twelve words in your
 mnemonic seed, they still have control over your account even if they do not know the password.
 
+### Reset Password
+
+:::warning
+
+Before following the instructions below make sure you have your mnemonic phrase stored in a safe place accessible to you.
+
+:::
+
+Let's say you created `ACCOUNT 1` protected by password `PSW 1`, and you also exported the JSON file `JSON 1`. To reset the passowrd of your polkadot `ACCOUNT 1` using the browser extension you must follow the following steps:
+
+- On the browser extension go to `ACCOUNT 1` and click "Forget account". This action will delete the access to your account. Note that your tokens are still in your account on the polkadot blockchain network.
+- On the browser extension click the "+" button in the topright corner and select the option "Import account from pre-existing seed". After entering the mnemonic phrase you can chose a new password `PSW 2`.
+
+:::info JSON files do not allow to change account passwords
+
+If you add the account to the extension using the option "Restore account from backup JSON file", this will allow you to restore access to your account using JSON file `JSON 1` protected by the password `PSW 1`, but not to set a new password. Thus, `PSW 1` will become the account password by default.
```

Comment:
```suggestion
If you add the account to the extension using the option "Restore account from backup JSON file", this will allow you to restore access to your account using JSON file protected by the password `PSW 1`, but does not let you set a new password. Thus, `PSW 1` will become the account password by default.
```